test/framework/suite.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python2
```

Comment:
please don't hide a change like this in a random PR, we should look into this in a dedicated PR...